doc/src/manual/types.md

```diff
@@ -410,6 +410,9 @@ There is much more to say about how instances of composite types are created, bu
 depends on both [Parametric Types](@ref) and on [Methods](@ref), and is sufficiently important
 to be addressed in its own section: [Constructors](@ref man-constructors).
 
+For most user-defined types `X`, you will want to define a method [`Base.broadcastable(x::X) = Ref(x)`](@ref man-interfaces-broadcasting)
```

Comment:
This seems too strong, as we don't follow that advice ourselves:
```julia
julia> ntypes = 0
0

julia> for name in names(Base; all=true)
           item = getfield(Base, name)
           if isa(item, DataType) && !isdefined(item, :instance)
               ntypes += 1
           end
       end

julia> ntypes
127

julia> length(methods(Base.broadcastable))
14
```
I'm not saying there shouldn't be more such definitions, but no one has needed them. Many types will never participate in broadcasting.